[Docker] Ninja build failed: error: call to function 'EraseIf'

I've used `docker-build.sh` but it failed at this step:

```
[15475/40414] CXX obj/components/domain_reliability/domain_reliability/google_configs.o
../../components/domain_reliability/google_configs.cc:16:8: warning: unused variable 'is_www_subdomain' [-Wunused-variable]
  bool is_www_subdomain =
       ^
1 warning generated.
[15488/40414] CXX obj/components/domain_reliability/domain_reliability/uploader.o
../../components/domain_reliability/uploader.cc:89:9: warning: code will never be executed [-Wunreachable-code]
        net::DefineNetworkTrafficAnnotation("domain_reliability_report_upload",
        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
../../components/domain_reliability/uploader.cc:80:9: note: silence by adding parentheses to mark code as explicitly dead
    if (true) {
        ^
        /* DISABLES CODE */ ( )
1 warning generated.
[15962/40414] CXX obj/net/net/url_request_test_job.o
FAILED: obj/net/net/url_request_test_job.o 
clang++ -MMD -MF obj/net/net/url_request_test_job.o.d -DUSE_UDEV -DUSE_AURA=1 -DUSE_GLIB=1 -DUSE_NSS_CERTS=1 -DUSE_X11=1 -DOFFICIAL_BUILD -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -DNO_UNWIND_TABLES -D_GNU_SOURCE -DCR_CLANG_REVISION=\"n353803-99ac9ce7-1\" -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D_FORTIFY_SOURCE=2 -D_LIBCPP_ABI_UNSTABLE -D_LIBCPP_DISABLE_VISIBILITY_ANNOTATIONS -D_LIBCXXABI_DISABLE_VISIBILITY_ANNOTATIONS -D_LIBCPP_ENABLE_NODISCARD -DCR_LIBCXX_REVISION=375504 -DNDEBUG -DNVALGRIND -DDYNAMIC_ANNOTATIONS_ENABLED=0 -DGLIB_VERSION_MAX_ALLOWED=GLIB_VERSION_2_40 -DGLIB_VERSION_MIN_REQUIRED=GLIB_VERSION_2_40 -DGOOGLE_PROTOBUF_NO_RTTI -DGOOGLE_PROTOBUF_NO_STATIC_INITIALIZER -DHAVE_PTHREAD -DNET_IMPLEMENTATION -DENABLE_BUILT_IN_DNS -DUSE_GIO -DU_USING_ICU_NAMESPACE=0 -DU_ENABLE_DYLOAD=0 -DUSE_CHROMIUM_ICU=1 -DU_ENABLE_TRACING=1 -DU_ENABLE_RESOURCE_TRACING=0 -DU_STATIC_IMPLEMENTATION -DICU_UTIL_DATA_IMPL=ICU_UTIL_DATA_FILE -DUCHAR_TYPE=uint16_t -I../.. -Igen -I../../third_party/protobuf/src -Igen/protoc_out -I../../third_party/protobuf/src -I../../third_party/boringssl/src/include -I../../third_party/zlib -I../../third_party/ced/src -I../../third_party/icu/source/common -I../../third_party/icu/source/i18n -I../../third_party/brotli/include -fprofile-sample-use=../../chrome/android/profiles/afdo.prof -fno-strict-aliasing --param=ssp-buffer-size=4 -fstack-protector -fno-unwind-tables -fno-asynchronous-unwind-tables -fPIC -pthread -fcolor-diagnostics -fmerge-all-constants -fcrash-diagnostics-dir=../../tools/clang/crashreports -Xclang -mllvm -Xclang -instcombine-lower-dbg-declare=0 -flto=thin -fsplit-lto-unit -fwhole-program-vtables -m64 -march=x86-64 -Xclang -fdebug-compilation-dir -Xclang . -no-canonical-prefixes -Wall -Wextra -Wimplicit-fallthrough -Wunreachable-code -Wthread-safety -Wextra-semi -Wno-missing-field-initializers -Wno-unused-parameter -Wno-c++11-narrowing -Wno-unneeded-internal-declaration -Wno-undefined-var-template -Wno-ignored-pragma-optimize -fno-omit-frame-pointer -g0 -ftrivial-auto-var-init=pattern -fsanitize=cfi-vcall -fsanitize-blacklist=../../tools/cfi/blacklist.txt -fsanitize=cfi-icall -fvisibility=hidden -Wheader-hygiene -Wstring-conversion -Wtautological-overlap-compare -O2 -fno-ident -fdata-sections -ffunction-sections -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include -Wno-shorten-64-to-32 -Wno-header-guard -I/usr/include/nss -I/usr/include/nspr -Wexit-time-destructors -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include -std=c++14 -fno-exceptions -fno-rtti -nostdinc++ -isystem../../buildtools/third_party/libc++/trunk/include -isystem../../buildtools/third_party/libc++abi/trunk/include -fvisibility-inlines-hidden -resource-dir=/usr/lib/llvm-8/lib/clang/8.0.0 -resource-dir=/usr/lib/llvm-8/lib/clang/8.0.0 -c ../../net/url_request/url_request_test_job.cc -o obj/net/net/url_request_test_job.o
In file included from ../../net/url_request/url_request_test_job.cc:5:
In file included from ../../net/url_request/url_request_test_job.h:13:
In file included from ../../net/url_request/url_request.h:23:
In file included from ../../net/base/auth.h:12:
In file included from ../../url/origin.h:19:
In file included from ../../base/strings/string_util.h:21:
../../base/stl_util.h:587:10: error: call to function 'EraseIf' that is neither visible in the template definition nor found by argument-dependent lookup
  return EraseIf(container, [&](const T& cur) { return cur == value; });
         ^
../../net/url_request/url_request_test_job.cc:181:9: note: in instantiation of function template specialization 'base::Erase<net::URLRequestTestJob *, std::__1::allocator<net::URLRequestTestJob *>, net::URLRequestTestJob *>' requested here
  base::Erase(g_pending_jobs.Get(), this);
        ^
../../base/stl_util.h:591:8: note: 'EraseIf' should be declared prior to the call site or in namespace 'net'
size_t EraseIf(std::list<T, Allocator>& container, Predicate pred) {
       ^
1 error generated.
[15969/40414] CXX obj/net/net/url_request_throttler_entry.o
ninja: build stopped: subcommand failed.

```

Any idea where to look at?

--- a/docker-stage1/Dockerfile
+++ b/docker-stage1/Dockerfile
@@ -1,13 +1,13 @@
-FROM ubuntu:18.04
+FROM ubuntu:20.04
 
-RUN apt-get update && apt-get -y upgrade
+RUN apt-get update --fix-missing && apt-get -y upgrade
 
 ADD scripts /scripts
 
 RUN /scripts/pre-build.sh
 
-RUN apt-get -y install git debhelper clang-8 lld-8 llvm-8-dev python python3 pkg-config ninja-build python-jinja2 ca-certificates gsettings-desktop-schemas-dev wget flex yasm xvfb wdiff gperf bison valgrind xz-utils x11-apps xfonts-base libglew-dev libgl1-mesa-dev libglu1-mesa-dev libegl1-mesa-dev libgles2-mesa-dev mesa-common-dev libxt-dev libre2-dev libgbm-dev libpng-dev libxss-dev libelf-dev libvpx-dev libpci-dev libcap-dev libdrm-dev libicu-dev libffi-dev libkrb5-dev libexif-dev libflac-dev libudev-dev libopus-dev libwebp-dev libxtst-dev libjpeg-dev libxml2-dev libgtk-3-dev libxslt1-dev liblcms2-dev libpulse-dev libpam0g-dev libsnappy-dev libavutil-dev libavcodec-dev libavformat-dev libglib2.0-dev libasound2-dev libjsoncpp-dev libspeechd-dev libminizip-dev libhunspell-dev libusb-1.0-0-dev libopenjp2-7-dev libmodpbase64-dev libnss3-dev libnspr4-dev libcups2-dev libevent-dev libjs-jquery libjs-jquery-flot libgcrypt20-dev libva-dev fonts-ipafont-gothic fonts-ipafont-mincho uuid-dev nodejs xcb-proto python-xcbgen && \
-  update-alternatives --install /usr/bin/clang clang /usr/bin/clang-8 800     && \
-  update-alternatives --install /usr/bin/clang++ clang++ /usr/bin/clang++-8 800 && \
-  update-alternatives --install /usr/bin/llvm-ar llvm-ar /usr/bin/llvm-ar-8 800 && \
-  update-alternatives --install /usr/bin/llvm-nm llvm-nm /usr/bin/llvm-nm-8 800
+RUN apt-get install -y apt-utils && apt-get -y --no-install-recommends install git debhelper clang-10 lld-10 llvm-10-dev python python3 pkg-config ninja-build python-jinja2 ca-certificates gsettings-desktop-schemas-dev wget flex yasm xvfb wdiff gperf bison valgrind xz-utils x11-apps xfonts-base libglew-dev libgl1-mesa-dev libglu1-mesa-dev libegl1-mesa-dev libgles2-mesa-dev mesa-common-dev libxt-dev libre2-dev libgbm-dev libpng-dev libxss-dev libelf-dev libvpx-dev libpci-dev libcap-dev libdrm-dev libicu-dev libffi-dev libkrb5-dev libexif-dev libflac-dev libudev-dev libopus-dev libwebp-dev libxtst-dev libjpeg-dev libxml2-dev libgtk-3-dev libxslt1-dev liblcms2-dev libpulse-dev libpam0g-dev libsnappy-dev libavutil-dev libavcodec-dev libavformat-dev libglib2.0-dev libasound2-dev libjsoncpp-dev libspeechd-dev libminizip-dev libhunspell-dev libusb-1.0-0-dev libopenjp2-7-dev libmodpbase64-dev libnss3-dev libnspr4-dev libcups2-dev libevent-dev libjs-jquery libjs-jquery-flot libgcrypt20-dev libva-dev fonts-ipafont-gothic fonts-ipafont-mincho uuid-dev nodejs xcb-proto python3-xcbgen python-setuptools libx11-xcb-dev libxcb-dri3-dev && \
+  update-alternatives --install /usr/bin/clang clang /usr/bin/clang-10 800 && \
+  update-alternatives --install /usr/bin/clang++ clang++ /usr/bin/clang++-10 800 && \
+  update-alternatives --install /usr/bin/llvm-ar llvm-ar /usr/bin/llvm-ar-10 800 && \
+  update-alternatives --install /usr/bin/llvm-nm llvm-nm /usr/bin/llvm-nm-10 800

